checkstyle.xml

```diff
@@ -90,7 +90,7 @@
     <!-- Modifier Checks                                    -->
     <!-- See http://checkstyle.sf.net/config_modifiers.html -->
     <module name="ModifierOrder"/>
-    <module name="RedundantModifier"/>
+    <!--<module name="RedundantModifier"/>-->

```

Comment:
We either need this commented out/removed, or we need to go back and remove a lot of instances of `public`. Once you fix these 9 instances, more pop-up.
```
[ant:checkstyle] [ERROR] /home/travis/build/square/picasso/picasso/src/main/java/com/squareup/picasso/Action.java:29:5: Redundant 'public' modifier. [RedundantModifier]
[ant:checkstyle] [ERROR] /home/travis/build/square/picasso/picasso/src/main/java/com/squareup/picasso/AssetRequestHandler.java:37:3: Redundant 'public' modifier. [RedundantModifier]
[ant:checkstyle] [ERROR] /home/travis/build/square/picasso/picasso/src/main/java/com/squareup/picasso/Dispatcher.java:460:5: Redundant 'public' modifier. [RedundantModifier]
[ant:checkstyle] [ERROR] /home/travis/build/square/picasso/picasso/src/main/java/com/squareup/picasso/MarkableInputStream.java:39:3: Redundant 'public' modifier. [RedundantModifier]
[ant:checkstyle] [ERROR] /home/travis/build/square/picasso/picasso/src/main/java/com/squareup/picasso/MarkableInputStream.java:43:3: Redundant 'public' modifier. [RedundantModifier]
[ant:checkstyle] [ERROR] /home/travis/build/square/picasso/picasso/src/main/java/com/squareup/picasso/NetworkRequestHandler.java:34:3: Redundant 'public' modifier. [RedundantModifier]
[ant:checkstyle] [ERROR] /home/travis/build/square/picasso/picasso/src/main/java/com/squareup/picasso/PicassoExecutorService.java:95:5: Redundant 'public' modifier. [RedundantModifier]
[ant:checkstyle] [ERROR] /home/travis/build/square/picasso/picasso/src/main/java/com/squareup/picasso/Stats.java:130:5: Redundant 'public' modifier. [RedundantModifier]
[ant:checkstyle] [ERROR] /home/travis/build/square/picasso/picasso/src/main/java/com/squareup/picasso/Utils.java:348:5: Redundant 'public' modifier. [RedundantModifier]
```